html-forms/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>HTML Forms</title>
+</head>
+<body>
+  <h1>Coding Form!</h1>
+  <form>
+    <label for="email">
+      Email
+      <div>
+        <input id="email" type="text" name="email">
+      </div>
+    </label>
+    <label for="reason">
+      Why is coding cool?
+      <div>
+        <select id="reason" name="reason">
+          <option value="default">Select an option</option>
+          <option value="interesting">Because it's interesting</option>
+          <option value="fun">Because it's fun</option>
+          <option value="powerful">Because it's powerful</option>
+        </select>
+      </div>
+    </label>
+    <label for="languages">
```

Comment:
This doesn't need to be a label, there should be a `label` element around each option's text